### PR TITLE
291: Show Coverage Over time of last four weeks and last four releases

### PR DESCRIPTION
**Pull request by zachmandeville**
PR for #291

### CODE REVIEW

File: ATTIC/deployment/local/Tiltfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# -*- mode: python; -*-
 k8s_yaml(kustomize('.'))
-docker_build('raiinbow/webapp', '../../../apps/webapp',
+docker_build('gcr.io/k8s-staging-apisnoop/webapp', '../../../apps/webapp',
```
Comment (hh) [inline ATTIC/deployment/local/Tiltfile:2]: Just wanting to note that this is what you want merged?

File: ATTIC/deployment/local/Tiltfile [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
   live_update=[
   fall_back_on(['package.json', 'package-lock.json']),
-  sync('apps/webapp','/src')
+  sync('apps/webapp','/src'),
+  run('touch src/client.js')
```
Comment (hh) [inline ATTIC/deployment/local/Tiltfile:6]: Is live update working?

Comment (zachmandeville) [reply ATTIC/deployment/local/Tiltfile:6]: unfortunately, I couldn't get it to work with our svelte app.

File: ATTIC/deployment/local/Tiltfile (hunk 1 above)
Comment (zachmandeville) [reply ATTIC/deployment/local/Tiltfile:2]: yah!  This is the standard place we are keeping the apisnoop docker images now, yah?

**Pull request merged**